packages/babel-plugin-transform-es2015-parameters/README.md

```diff
@@ -2,6 +2,12 @@
 
 > Compile ES2015 default and rest parameters to ES5
 
+This plugin transforms ES2015 parameters to ES5, this includes:
```

Comment:
Deleted from the website repo, in https://github.com/babel/babel.github.io/pull/1135/files#diff-ddf60b2ede8d9cc71fef79e5aebbcbaf